python/ray/serve/api.py

```diff
@@ -247,6 +244,8 @@ def create_backend(func_or_class,
         class CustomActor(RayServeMixin, func_or_class):
             @wraps(func_or_class.__init__)
             def __init__(self, *args, **kwargs):
+                # Initialize serve so it can be used in backends.
+                init()
```

Comment:
i thought i already did that somewhere... possible got lost in a dev branch. thanks for the fix!

Reply:
I actually removed it because I mistakenly thought it wasn't necessary after my changes :)